app/scripts/config.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+], function () {
+  'use strict';
+
+  var Config = {
+    oauth: {
+      authEndpoint: '{%= config.get("server.oauth.authEndpoint") %}'
```

Comment:
Do we need the entire config object here, or do you just want to pass the `authEndpoint`?